Godeps/Godeps.json

```diff
@@ -1932,6 +1932,38 @@
 			"Rev": "6b7015e65d366bf3f19b2b2a000a831940f0f7e0"
 		},
 		{
+			"ImportPath": "github.com/containers/image/signature",
+			"Rev": "c07f8fdceeda1517556602778a61ba94894e7c02"
+		},
+		{
+			"ImportPath": "github.com/containers/image/types",
+			"Rev": "c07f8fdceeda1517556602778a61ba94894e7c02"
+		},
+		{
+			"ImportPath": "github.com/containers/image/docker/reference",
+			"Rev": "c07f8fdceeda1517556602778a61ba94894e7c02"
+		},
+		{
+			"ImportPath": "github.com/containers/image/docker/policyconfiguration",
+			"Rev": "c07f8fdceeda1517556602778a61ba94894e7c02"
+		},
+		{
+			"ImportPath": "github.com/containers/image/manifest",
+			"Rev": "c07f8fdceeda1517556602778a61ba94894e7c02"
+		},
+		{
+			"ImportPath": "github.com/containers/image/transports",
+			"Rev": "c07f8fdceeda1517556602778a61ba94894e7c02"
+		},
+		{
+			"ImportPath": "github.com/containers/image/version",
+			"Rev": "c07f8fdceeda1517556602778a61ba94894e7c02"
+		},
+		{
+			"ImportPath": "github.com/containers/storage/pkg/homedir",
```

Comment:
It's not about hating you, I ❤️ you man! It's about having clear history when and what was added 😉